code/API_definitions/edge-discovery-service.yaml

```diff
@@ -11,14 +11,198 @@ info:
   description: |
     # Introduction
     ---
-    Use the Edge Discovery API for
-    discovery of the most optimal edge cloud zone for a specific user as well as specific application instance the user is trying to connect to.
+    The Edge Discovery API is designed to help users discover the most optimal
+    edge cloud zones for their applications. This API allows users to find the
+    best regions and zones for their edge cloud applications, discover optimal
+    edge cloud zones for deployed applications, and find the best application
```

Comment:
I suppose this intent "discover optimal edge cloud zones for deployed applications" is valid for the end users which would connect with the edge applications instances. In that case we need to possibly indicate the different type of users who may consume these APIs.